### PR TITLE
CopterHack-2022: V-NAV

### PR DESCRIPTION
**Pull request by vas0x59**

### CODE REVIEW

File: docs/ru/v-nav.md [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+
+[CopterHack-2022](copterhack2022.md), команда V-NAV.
+
+## Информация о команде
+
+Состав команды:
+
+- Артём Баталов, [@bart02](http://t.me/bart02), программист
+- Василий Юрьев, [@vasily.yuryev](https://t.me/vasily_yuryev), программист
+- Никита Локтев, [@nikilokser](https://t.me/nikilokser) , инженер
```
Comment (okalachev) [inline docs/ru/v-nav.md:11]:
```suggestion
- Артём Баталов, [@bart02](http://t.me/bart02), программист.
- Василий Юрьев, [@vasily.yuryev](https://t.me/vasily_yuryev), программист.
- Никита Локтев, [@nikilokser](https://t.me/nikilokser) , инженер.
```

File: docs/ru/v-nav.md [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+
+## Описание проекта
+
+### Идея проекта
+
+На данный момент полеты в indoor по неподготовленным помещениям сложны из-за необходимости вмешательства в среду, и настройки полётной зоны в ПО БПЛА. Также имеются сложности в организации outdoor полетов, где невозможно использование глобальных общедоступных спутниковых систем позиционирования (GNSS), например для мониторинга инфраструктуры предприятий, таких как, нефтеперерабатывающие заводы, электрические подстанции.
+
+Таким образом, мы ставим задачу обеспечить возможность полётов в указанных средах. Одним из способов решения данной задачи является применение оптической системы навигации с возможностью построения 3D карты местности (visual SLAM).
+
+В качетве сенсоров планируется использовать массив оптических камер, обеспечивающих охват местности во всех направлениях, и IMU сенсор.
```
Comment (okalachev) [inline docs/ru/v-nav.md:21]:
```suggestion
В качестве сенсоров планируется использовать массив оптических камер, обеспечивающих охват местности во всех направлениях, и IMU-сенсор.
```

**Comment by SeliverstovaE** [comment]
Dear friend,

Thank you for your interest in the CopterHack 2022 contest. Your application has successfully passed the experts’ review and was accepted for the second project stage of the competition, which will be held until February 28, 2022. This stage will be followed by monthly updates, lectures, and guidance from industry experts.

What's next?

1. Join the Telegram group with participants of the second project stage “[CopterHack 2022, participants chat](https://t.me/joinchat/1KIDu8p5FGhlYTBi )”.  
2. Every month, you're expected to prepare free-form reports on the progress of your project (update) in this draft pull request. In your report please indicate your progress, what tasks have been solved. You are encouraged to attach photos and videos that demonstrate the results of the work done within the month.
3. Send the link to the update to the participants chat in the Telegram group.

In response to the message, you will receive feedback from the competition jury.

There is no specific date for sending the report, so you can send the report between the beginning and the end of the month.

Monthly updates become mandatory from November 2021. But as soon as you are in the telegram group, you can already send updates and receive feedback from the jury.

We are interested in helping you to complete your project as much as possible by the final stage. We wish you success!

Always in touch! Your CopterHack team.

**Comment by SeliverstovaE** [comment]
Dear V-NAV team,
We are closing pull request for your project as you didn’t get to the final round of the competition. 

You are welcome to submit your article to the clover documentation as an clover platform-based project. In order to do this please create a new pull request or make a comment to the existing one.

We wish you success in implementing your project and invite you to the next events.

**Pull request closed without merging**